templates/k-bims-ledet/0/docker-compose.yml

```diff
@@ -0,0 +1,362 @@
+
+  nginx-conf-data:
+    image: kartoza/ledet_nginx_conf
+    restart: unless-stopped
+    labels:
+      io.rancher.container.start_once: true
+    volumes:
+      - nginx-conf-data:/etc/nginx/sites-available
+
+  lb:
+    image: rancher/lb-service-haproxy:v0.7.15
+    ports:
+    - ${WEBSERVER_PORT}:80/tcp
+    labels:
+      io.rancher.container.agent.role: environmentAdmin,agent
+      io.rancher.container.agent_service.drain_provider: 'true'
+      io.rancher.container.create_agent: 'true'
+
+  web:
+    image: nginx
+    volumes:
+    - django-static:/home/web/static:ro
+    - django-media:/home/web/media:ro
+    - nginx-conf-data:/etc/nginx/sites-available
+    - nginx-logs:/var/log/nginx
+    entrypoint: /etc/nginx/sites-available/docker-entrypoint.sh
+    links:
+      - uwsgi:uwsgi
+      - geoserver:geoserver
+    command: prod
+    labels:
+      io.rancher.container.pull_image: always
+    restart: unless-stopped
+
+  data-dir-conf:
+    image: geosolutionsit/geoserver_data:2.13.x
+    restart: unless-stopped
+    labels:
+      io.rancher.container.start_once: true
+    volumes:
+      - geoserver-data-dir:/geoserver_data/data
+
+  geoserver:
+    image: geosolutionsit/geoserver-docker:2.13.x
+    restart: unless-stopped
+    links:
+      - db:db
+      - data-dir-conf:data-dir-conf
+    volumes:
+      - geoserver-data-dir:/geoserver_data/data
+    environment:
+      SITEURL: http://${SITE_URL}/
+      GEOSERVER_ADMIN_PASSWORD: ${GEOSERVER_ADMIN_PASSWORD}
+
+  elasticsearch:
+    image: elasticsearch:2.4.2
+    environment:
+      cluster.name: docker-cluster
+      bootstrap.memory_lock: 'true'
+    labels:
+      io.rancher.container.pull_image: always
+
+  uwsgi: &uwsgi
+    image: kartoza/ledet_uwsgi:latest
+    environment:
+      DATABASE_HOST: db
+      DATABASE_NAME: gis
+      DATABASE_PASSWORD: docker
+      DATABASE_USERNAME: docker
+      DJANGO_SETTINGS_MODULE: ledet_core.settings.prod_docker
+      RABBITMQ_HOST: rabbitmq
+      VIRTUAL_HOST: healthyrivers.kartoza.com
+      VIRTUAL_PORT: '8080'
+      SITE_DOMAIN_NAME: http://${SITE_URL}
+      SITEURL: http://${SITE_URL}/
+      PYTHONPATH: /usr/src/app:/usr/src/bims:/home/web/django_project
+      HAYSTACK_HOST: elasticsearch
+      GEOCONTEXT_URL: ${GEOCONTEXT_URL}
+      CONTACT_US_EMAIL: ${ADMIN_EMAIL}
+      ROOT_URLCONF: ledet_core.urls
+      GEOSERVER_PUBLIC_LOCATION: http://${SITE_URL}/geoserver/
+      STATIC_ROOT: /home/web/static
+      GEONODE_DATABASE: gis
+      GEONODE_DATABASE_USER: docker
+      MEDIA_ROOT: /home/web/media
+      ADMIN_EMAILS: ${ADMIN_EMAIL}
+      GEOSERVER_ADMIN_PASSWORD: ${GEOSERVER_ADMIN_PASSWORD}
+      GEOIP_PATH: /home/web/media/geoip.db
+      GEONODE_GEODATABASE_PASSWORD: geonode_data
+      GEONODE_INSTANCE_NAME: geonode
+      DEFAULT_BACKEND_DATASTORE: datastore
+      IS_CELERY: 'False'
+      ALLOWED_HOSTS: '["localhost","127.0.0.1","${SITE_URL}"]'
+      GEOSERVER_LOCATION: http://geoserver:8080/geoserver/
```

Comment:
Maybe just add a comment here explaining this references the linked geoserver container